src/utils/stacktrace.ts

```diff
@@ -0,0 +1,95 @@
+import ErrorStackParser from "error-stack-parser";
+
+export type RawStackFrames = string;
+
+const getErrorTitle = (e: Error): string => `${e.name || "Error"}: ${e.message}`;
+const getErrorStackFrames = (e: Error): string | undefined => e.stack?.replace(getErrorTitle(e) + "\n", "");
```

Comment:
CT could lose `e.name` due to serialization

Reference: https://nodejs.org/docs/latest-v16.x/api/errors.html#errorcapturestacktracetargetobject-constructoropt

> The first line of the trace will be prefixed with ${myObject.name}: ${myObject.message}.

Reply:
Heh, first time i see `JSON.stringify` with array being used as the second argument.